Lint warning: yamlint warning  line too long

### What did you have in mind?

Release CI flow as a line which is too long that creates an error when running `yamlint`

### Are you trying to fix a problem?

When running poe lint the `yamlint` step reports a warning:
```txt
Poe => poetry run yamllint --format colored --strict .
.\.github\workflows\release.yml
  147:101   warning  line too long (101 > 100 characters)  (line-length)
```

### Any lead on how this feature can be implemented?

_No response_

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -144,7 +144,7 @@ jobs:
           oauth_token: ${{ secrets[needs.secrets_keys.outputs.twitter_access_token] }}
           oauth_token_secret: ${{ secrets[needs.secrets_keys.outputs.twitter_access_token_secret] }}
           message: |
-            🚀🎉 Aioswitcher ${{ steps.bumper.outputs.new_version  }} has been released! 🎉🚀
+            🚀🎉 Aioswitcher ${{ steps.bumper.outputs.new_version }} has been released! 🎉🚀
             ${{ steps.gh_release.outputs.html_url }}
 
             #aioswitcher #smarthome #python

